### DISCUSSION TITLE
Documentation search hangs up.

### DISCUSSION THREAD

**Discussion by pauljurczak** [beeware/toga#1472] [Q&A]
I've heard that Toga plays nice with matplotlib. I wanted to find more details by searching for `matplotlib` on https://toga.readthedocs.io/en/latest/index.html. Unfortunately, that didn't produce any results after 5 minutes wait. Any links on matplotlib use with Toga will be appreciated.

**Comment by freakboy3742** [comment]
"Plays nice" is possibly a little generous, but it does play :-) 

The state of support (such as it is) is that there is a [toga-chart widget](https://github.com/beeware/toga-chart) that provides a matplotlib charting backend using native Toga widgets. 

It's very much beta at the moment; but it's been known to work on macOS, Windows and Linux. There's an example app in the `examples` folder of that repository that shows how it can be used.
